erts/emulator/beam/erl_db.c

```diff
@@ -1203,35 +1203,29 @@ BIF_RETTYPE ets_take_2(BIF_ALIST_2)
     BIF_RET(ret);
 }
 
-/* 
-** update_element(Tab, Key, {Pos, Value})
-** update_element(Tab, Key, [{Pos, Value}])
-*/
-BIF_RETTYPE ets_update_element_3(BIF_ALIST_3)
+static BIF_RETTYPE do_update_element(Process *p, DbTable *tb,
+		Eterm arg2, Eterm arg3, Eterm arg4)
```

Comment:
Give the arguments real names, something like `key`, `pos_val`, `default_obj`.

Reply:
Modeled on `do_update_conter`. I'll see to it :+1:

Reply:
Done in last commit.